projects/ngrx.io/content/guide/signals/signal-store/testing.md

```diff
@@ -0,0 +1,687 @@
+<!-- TOC -->
+  * [Introduction](#introduction)
+    * [On Testing in general](#on-testing-in-general)
+    * [What to test](#what-to-test)
+    * [TestBed or not?](#testbed-or-not)
+  * [Testing the Signal Store](#testing-the-signal-store)
+    * [Globally provided](#globally-provided)
+    * [Locally provided](#locally-provided)
+    * [`withComputed`](#withcomputed)
+    * [`withMethods`, DI, and asynchronous tasks](#withmethods-di-and-asynchronous-tasks)
+    * [`rxMethod`](#rxmethod)
+      * [with Observables](#with-observables)
+      * [with Signals](#with-signals)
+    * [Custom extensions](#custom-extensions)
+  * [Mocking the Signal Store](#mocking-the-signal-store)
+    * [Native Mocking](#native-mocking)
+    * [ng-mocks](#ng-mocks)
+    * [Partial Mocking via Spies](#partial-mocking-via-spies)
+<!-- TOC -->
+
+This is still a draft. Markdown is used for better readability. The rendered version of this guide is available at https://github.com/rainerhahnekamp/ngrx/blob/docs/signals/testing/projects/ngrx.io/content/guide/signals/signal-store/testing.md.
+
+Once the content is finalized, we will replace markdown with `<code>` tags.
+
+The examples used in this guide are available at https://github.com/rainerhahnekamp/ngrx-signal-store-testing
+
+## Introduction
+
+### On Testing in General
```

Comment:
What are your thoughs on adding an "integration testing" section here (or as a recipe)?
This would test the component consuming the signal store.

Similar to https://ngrx.io/guide/store/testing

Reply:
Yes of course. Will do that.